projects/core/src/routing/configurable-routes/config/default-storefront-routes-config.ts

```diff
@@ -8,7 +8,10 @@ export const defaultStorefrontRoutesConfig: RoutesConfig = {
   register: { paths: ['register'] },
   resetPassword: { paths: ['login/pw/change'] },
   forgotPassword: { paths: ['forgot-password'] },
-  checkout: { paths: ['checkout'] },
+  shippingAddress: { paths: ['checkout/shipping-address'] },
+  deliveryMode: { paths: ['checkout/delivery-mode'] },
+  paymentDetails: { paths: ['checkout/payment-details'] },
+  reviewOrder: { paths: ['checkout/review-order'] },
   orderConfirmation: { paths: ['order-confirmation'] },
```

Comment:
We need to figure out how we can combine our CheckoutConfig routes from defaultCheckoutConfig with this file to setup routing dynamically. This config file is on a core side.

Reply:
For now i think we can keep it mocked.

Reply:
This modification refers to following breaking change: https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/1994